mapit_mysociety_org/middleware.py

```diff
@@ -0,0 +1,17 @@
+from .multidb import use_primary
+
+READ_METHODS = frozenset(['GET', 'HEAD', 'OPTIONS', 'TRACE'])
+
+
+def force_primary_middleware(get_response):
```

Comment:
It might be helpful for future reference to include a comment outlining why this middleware does what it does. I read it in the context of your explanation on standup today, but without that I might have struggled to understand the reason for the short-lived cookie.